components/brex/sources/new-transfer-event/new-transfer-event.mjs

```diff
@@ -0,0 +1,16 @@
+import common from "./common.mjs";
+import brexApp from "../../brex.app.mjs";
+
+export default {
+  ...common,
+  dedupe: "unique",
+  type: "source",
+  key: "brex-new-transfer-event",
+  name: "New Transfer Event",
```

Comment:
Since this source uses a webhook, it should have "(Instant)" at the end of the name.
```suggestion
  name: "New Transfer Event (Instant)",
```